specification.md

```diff
@@ -345,7 +345,7 @@ The second way would be to directly filter only the "veggie like" vegetables wit
 ```json
 {
   "stateDataFilter": {
-    "input": "${ {vegetables: .vegetables[] | select(.veggieLike == true)} }"
+    "input": "${vegetables: .vegetables[] | select(.veggieLike == true)}"
```

Comment:
the expression that was removed is a clear jq expression you can cut/paste into jq play for example
you are taking away users ability to use 1/2 of the jq spec like this imo

Reply:
alo wrong, yes. replacement error